.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.11.4-otp-23
-erlang 23.2.7
+elixir 1.14.2-otp-25
+erlang 25.1.2
```

Comment:
I know it is not the latest, but let's keep `erlang 25.1.1` and `elixir 1.14.0-otp-25` which is what we already using elsewhere.